kinetic/distribution.yaml

```diff
@@ -7581,6 +7581,15 @@ repositories:
       type: git
       url: https://github.com/evocortex/optris_drivers.git
       version: kinetic-devel
+  orb_slam2_ros:
+    doc:
+      type: git
+      url: https://github.com/appliedAI-Initiative/orb_slam_2_ros.git
```

Comment:
You're missing a version tag here. Usually you put the development branch so it gets updated on new changes.